change/@ni-spright-components-08e968ac-291f-4c66-949e-115ca77deef2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Create initial spright-components package",
+  "packageName": "@ni/spright-components",
+  "email": "7282195+m-akinc@users.noreply.github.com",
+  "dependentChangeType": "none"
+}
```

Comment:
nit: I got thrown off by this statement in the PR description.
>Storybook for spright-components will be a separate submission.

There's a bunch of storybook stuff in this PR. No point in deleting it just to re-add it in a future PR, but could you make the PR description more accurately reflect the infrastructure that's being added?